packages/playground/src/components/common-breadcrumb/common-breadcrumb.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
```

Comment:
Should we make a context for the common breadcrumbs to hold each state? 
@srdjan-harness

Reply:
Logic and state is in Header component. Gonna check if we need it.